index.html

```diff
@@ -665,6 +665,7 @@
         [[!SERVICE-WORKERS]], which this specification extends.
       </p>
       <pre class="idl">
+        [SecureContext]
```

Comment:
I'm really confused by the specification of this:

> Any extended attribute specified on a partial interface definition is considered to appear on the interface itself.

But then the text later says that the attributes only apply to the partial interface.  In this case, `ServiceWorkerRegistration` is annotated in the same way, so this is OK (it's not in the notifications API though).